code/modules/mob/living/carbon/life.dm

```diff
@@ -314,7 +314,10 @@
 /mob/living/carbon/handle_status_effects()
 	..()
 	if(staminaloss)
-		adjustStaminaLoss(-3)
+		if(IsKnockdown() || IsUnconscious())

```

Comment:
Then the comments in the life file are pretty outdated:

"//this updates all special effects: stun, sleeping, knockdown, druggy, stuttering, etc.."